windows/QMK Toolbox/HidConsole/HidConsoleListener.cs

```diff
@@ -0,0 +1,136 @@
+﻿using HidLibrary;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management;
+
+namespace QMK_Toolbox.HidConsole
+{
+    public class HidConsoleListener
```

Comment:
If it's got a `Dispose()` you probably want to make this a normal `IDisposable` pattern with the finaliser and GC suppression?

Reply:
I understood some of those words.

Like this?
![image](https://user-images.githubusercontent.com/4781841/130732896-1b727030-1620-43e7-bb69-178c94b3985b.png)